tox.ini

```diff
@@ -11,7 +11,7 @@ deps=
 extras=
     dataflow_graphic
 # Set the environment variable INMANTA_EXTRA_PYTEST_ARGS='--fast' to run in fast mode
-commands=py.test --log-level DEBUG --cov=inmanta --cov-report term --cov-report xml --junitxml=junit-{envname}.xml -vvv {env:INMANTA_EXTRA_PYTEST_ARGS:} --durations=50 tests/
+commands=py.test --log-level DEBUG --log-format="%(asctime)s.%(msecs)03d %(levelname)s %(message)s" --cov=inmanta --cov-report term --cov-report xml --junitxml=junit-{envname}.xml -vvv {env:INMANTA_EXTRA_PYTEST_ARGS:} --durations=50 tests/
```

Comment:
The ticket suggests to add this setting to the `pytest.ini` file to enable this settings for all pytest runs, because this way the setting is only enabled when running the tests using tox. It see it's also possible to configure pytest settings using the `setup.cfg`. I think that's even better, because that way we don't need to introduce yet another config file. @sanderr Do you agree. I see you added the suggestion for the `pytest.ini` file.

Reply:
Agreed. And thanks for showing me you can inspect edit history, that's very useful.